dosdp/.htaccess

```diff
@@ -0,0 +1,9 @@
+Options +FollowSymLinks
+RewriteEngine on
+
+
+# types --> includes/types
+RewriteRule ^spec(\/?)$ spec/DOSDP_schema_full.json
```

Comment:
Is the intent this would redirect to the below rule?  That resources doesn't exist in that repo.  And also if you are trying to redirect to raw JSON, you may need a github pages or similar URL rather than the github UI below.

Reply:
Changed to a few static URLs, which is all we need right now.